src/sync_template.sh

```diff
@@ -36,14 +36,19 @@ NEW_BRANCH="${PR_BRANCH_NAME_PREFIX}_${NEW_TEMPLATE_GIT_HASH}"
 
 echo "::group::Check new changes"
 echo "::debug::new Git HASH ${NEW_TEMPLATE_GIT_HASH}"
-if [ -r ${TEMPLATE_VERSION_FILE_NAME} ]
-then
+
+if [[ -f "$TEMPLATE_VERSION_FILE_NAME" ]]; then
```

Comment:
Small question: In #249 it is mentioned that the `.github/` folder should be the first lookup path.
Reading this `if/else` it is not consistent also to next section with the `TEMPLATE_VERSION_FILE_NAME`

Reply:
I'm not sure I understood your point, sorry.

Reply:
The idea is to use the files as if they where inside `.github` folder by Default:
1. Use the files if they are located inside `.github` folder (Default use case)
2. If they are not located inside `.github` BUT are located inside root folder, use them from root folder. (Use case where people used this Action before this Feature)
3. If the files are not located neither on `.github` nor on root folders, use them as if they where inside `.github`. (Use case where a fresh usage of this Action)

Reply:
> The idea is to use the files as if they where inside `.github` folder by Default:
> 
> 1. Use the files if they are located inside `.github` folder (Default use case)
> 2. If they are not located inside `.github` BUT are located inside root folder, use them from root folder. (Use case where people used this Action before this Feature)
> 3. If the files are not located neither on `.github` nor on root folders, use them as if they where inside `.github`. (Use case where a fresh usage of this Action)

Yes :) but if I read the algorithm (if/else) then it first searches in the repo root, afterwards (if not found else) in the `.github/` folder. If someone has 2 files (one in root dir, the other one in `.github/`) it will take the one within root.

So I assume a change of the order in if/else would do the trick here